certified-connectors/Yarado/apiDefinition.swagger.json

```diff
@@ -0,0 +1,165 @@
+	  "email": "info@yarado.com"
+	}  
+  },
+  "host": "api.yarado.com",
+  "basePath": "/",
+  "x-ms-connector-metadata": [
+    {
+	  "propertyName": "Website",
+	  "propertyValue": "https://yarado.com"
+	},
+	{
+	  "propertyName": "Privacy policy",
+	  "propertyValue": "https://www.yarado.com/terms"
+	}, 
+	{
+	  "propertyName": "Categories",
+	  "propertyValue": "Business Management;Productivity"
+	}
+  ],
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/v1/robots": {
+      "get": {
+        "summary": "GET Robots",
+        "description": "GET Robots",
+        "operationId": "GetRobots",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "id": {
+                    "type": "string"
+                  },
+                  "name": {
+                    "type": "string"
+                  },
+                  "organisation_name": {
+                    "type": "string"
+                  },
+                  "is_deleted": {
+                    "type": "boolean"
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/v1/task-files": {
+      "get": {
+        "summary": "GET Task Files",
+        "description": "GET Task Files",
+        "operationId": "GetTaskFiles",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "id": {
+                  "type": "string"
+                  },
+                  "filename": {
+                    "type": "string"
+                  },
+                  "created_by": {
+                    "type": "string"
+                  },
+                  "created_at": {
+                    "type": "string"
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/v1/task-schedules": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "robot_id": {
+                  "type": "string",
+                  "description": "robot_id",
+                  "title": "robot_id"
+                },
+                "task_file_id": {
+                  "type": "string",
+                  "description": "task_file_id",
```

Comment:
@MarcinKowalczukYarado make sure that descriptions and titles look like sentences, they need to be grammatically correct and free of typographical errors. More information can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).